punishments.js

```diff
@@ -18,10 +18,12 @@ let fs = require('fs');
 let path = require('path');
 
 const PUNISHMENT_FILE = path.resolve(__dirname, 'config/punishments.tsv');
+const ROOMBANS_FILE = path.resolve(__dirname, 'config/roombans.tsv');
```

Comment:
You do this thing where you look at global punishments named `PUNISHMENT` and name your room punishments `ROOMBANS`. Why not `PUNISHMENT` and `ROOM_PUNISHMENT`?